src/main/scala/com.gu.gibbons/UserDidNotAnswer.scala

```diff
@@ -26,23 +26,17 @@ class UserDidNotAnswer[F[_] : Monad](settings: Settings, email: EmailService[F],
       * 3- Delete those keys
       * 4- Send an email to inform users their keys have been deleted
       */
-    def run(dryRun: Boolean): F[Result] = 
-      if (dryRun)
-        for {
-          _ <- logger.info(s"Getting all the users which have not been extended since ${Settings.gracePeriod}")
-          users <- bonobo.getInactiveUsers(Settings.gracePeriod)
-        } yield DryRun(users)
-      else
-        for {
-          _ <- logger.info(s"Getting all the users which have not extended their account since ${Settings.gracePeriod}")
-          users <- bonobo.getInactiveUsers(Settings.gracePeriod)
-          _ <- logger.info(s"Found ${users.length} users. Let's delete these keys...")
-          ress <- users.filterNot(u => Settings.whitelist(u.id.id)).traverse { user =>
-            for {
-              _ <- bonobo.deleteUser(user)
-              res <- email.sendDeleted(user)
-            } yield user.id -> res
-          }.map(_.toMap)
-          _ <- logger.info("That's a wrap! See ya.")
-        } yield FullRun(ress)
+    def run(dryRun: Boolean): F[Map[UserId, EmailResult]] = 
```

Comment:
Nothing interesting here, I just moved the `dryRun` test inside the program to get rid of `Result`.